pkg/util/limiter/query_limiter.go

```diff
@@ -15,25 +16,32 @@ import (
 type queryLimiterCtxKey struct{}
 
 var (
-	ctxKey          = &queryLimiterCtxKey{}
-	errMaxSeriesHit = "The query hit the max number of series limit (limit: %d)"
+	ctxKey              = &queryLimiterCtxKey{}
+	errMaxSeriesHit     = "The query hit the max number of series limit (limit: %d)"
+	errMaxChunkBytesHit = "The query hit the aggregated chunks size limit (limit: %d)"
```

Comment:
You can export it so you can use it in `TestBlocksStoreQuerier_Select` instead of copy-pasting the error string there. Same can be done for `errMaxSeriesHit`.

As already suggested in a previous comment, I would make explicit the unit:

```suggestion
	errMaxChunkBytesHit = "The query hit the aggregated chunks size limit (limit: %d bytes)"
```